src/app/hooks.ts

```diff
@@ -21,10 +22,24 @@ export const zoneSelector = createDraftSafeSelector(
   (state) => state
 );
 
+export const selectZonesWithoutStatus = (state: RootState): Zone[] =>
```

Comment:
could you add a comment to explain why and what the method is doing exactly?
I guess something like "retrieve zones parameters not checking status to avoid unnecessary refresh is status changed"
I am not totally sure of the naming, I thought initially that we wanted the zone that had no status. But I did not find a better one so a comment may be enough

Reply:
actually even only calling it zone like in src/components/recommandations/RecoReport.tsx may be ok